db/bigtable.go

```diff
@@ -1542,279 +1831,400 @@ func (bigtable *Bigtable) SaveValidatorIncomeDetails(epoch uint64, rewards map[u
 	return nil
 }
 
-func (bigtable *Bigtable) GetEpochIncomeHistoryDescending(startEpoch uint64, endEpoch uint64) (*itypes.ValidatorEpochIncome, error) {
-	ctx, cancel := context.WithDeadline(context.Background(), time.Now().Add(time.Second*30))
-	defer cancel()
+func (bigtable *Bigtable) MigrateEpochSchemaV1ToV2(epoch uint64) error {
```

Comment:
**Question:** This function contains some error logs that do not stop the function. Personally I think a zero error policy should be applied. Here is an example:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/ef0180fc0f0250c8c83a06986f59a5affa18e5c3/db/bigtable.go#L1874
What are your thoughts on that?

Reply:
this was a leftover comment from a debugging session, can be deleted